Automattic/wp-calypso
# Site Assembler - Rename category Featured as All to explore all patterns

Related to https://github.com/Automattic/wp-calypso/issues/77129 #76451 #76580 

## Proposed Changes

* Rename the category `Featured` as `All` in the assembler only 
* Update the description
* Enable feature flag in staging and production

This is not in the editor because there is a performance issue when rendering a category with thousands of patterns. See https://github.com/Automattic/wp-calypso/issues/77129

<img width="698" alt="Screenshot 2566-05-19 at 18 58 18" src="https://github.com/Automattic/wp-calypso/assets/1881481/bca79479-9a71-414b-a163-4e311547516b">


## Testing Instructions



- Create a new site with any plan
- Continue until the Design Picker
- Scroll down and click `Start designing`
- Click `Homepage` > `Add patterns` 
- Verify that the category `All` is the first and contains all patterns
- Verify that there aren't performance issues when scrolling all the patterns

## Pre-merge Checklist
